docs/serverless/ingest/auto-import.mdx

```diff
@@ -0,0 +1,87 @@
+
+While Elastic has 400+ [prebuilt data integrations](((integrations-docs))), Automatic Import helps you extend data coverage to other security-relevant technologies and applications. Elastic integrations (including those created by Automatic Import) normalize data to [the Elastic Common Schema (ECS)](https://www.elastic.co/guide/en/ecs/current/ecs-reference.html), which creates uniformity across dashboards, search, alerts, machine learning, and more. 
+
+
+<DocCallOut title="Tip">
+Click [here](https://elastic.navattic.com/automatic-import) to watch a demo of how Automatic Import works before setting it up yourself.
+</DocCallOut>
+
+<DocCallOut title="Requirements">
+
+- A working <DocLink slug="/serverless/security/connect-to-bedrock" text="Amazon Bedrock connector"/>. Automatic Import currently works with all variants of Claude 3. In this technical preview, other models are not yet supported.
+- A [Security Analytics Complete](https://www.elastic.co/pricing/serverless-security) subscription.
+- A sample of the data you want to import in JSON or NDJSON format. 
+
+</DocCallOut>
+
+<DocCallOut color="warning" title="Important">
+Using Automatic Import allows our customers to create new third-party data integrataions through the use of third-party generative AI models (“GAI models”). Any third-party GAI models that you may choose to use are owned and operated by their respective providers. Elastic does not own or control these third-party GAI models, nor does it influence their design, training or data handling practices. Using third party GAI models with Elastic solutions, and using your data with third-party GAI models is at your discretion. Elastic bears no responsibility or liability for the content, operation, or use of these third-party GAI models, nor for any potential loss or damage arising from their use. Users are advised to exercise caution when using GAI models with personal, sensitive, or confidential information, as data submitted may be used for training of the models or other purposes. Elastic recommends familiarizing yourself with the development practices and terms of use of any third-party GAI models before use.
+
+You are responsible for ensuring that your use of Automatic Import complies with the terms and conditions of any third-party platform you connect with.
+</DocCallOut>
+
+
+## Create a new custom integration
+
+1. In ((elastic-sec)), click **Add integrations**.
+2. Under **Can't find an integration?** click **Create new integration**.
+
+<DocImage url="images/auto-import-create-new-integration-button.png" alt="The Integrations page with the Create new integration button highlighted" />
```

Comment:
Could we indent the images under their relevant steps? The steps seem a bit disconnected with the current indentation.